dts/bindings/gpio/renesas,ra8-gpio.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas RA8 GPIO
+
+compatible: "renesas,ra8-gpio"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  port:
```

Comment:
This is able to determine by address offset

Reply:
@soburi : Thank you for your advice. We saw your method for this case.
First, may I confirm that there is any special improvement when change to use this method?
Because we are supporting for new board, we will get information from HWM to update for dts (include port information is OK). When change to calculation offset method, can it impact to the latency?

Reply:
@soburi : Thank you very much for your comment. We understood. Because it's not blocking, let's us keep as current implementation for easy checking with HWM.